app/build.gradle

```diff
@@ -36,7 +36,10 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation 'com.google.ar.sceneform:filament-android:1.17.1'
```

Comment:
Why do you add this import? Remove it please

Reply:
Fixed